src/amber/cli/commands/database.cr

```diff
@@ -52,17 +52,19 @@ module Amber::CLI
             `crystal db/seeds.cr`
             Micrate.logger.info "Seeded database"
           when "migrate"
-            Micrate::Cli.run_up
+            # TODO(kevinsjoberg): Remove begin/rescue as Micrate properly
+            # handles empty migration directories.
+            begin
+              Micrate::Cli.run_up
+            rescue e : IndexError
+              exit! "no migrations to run"
+            end
```

Comment:
I would prefer an if statement over expecting an `IndexError` to be raised.
```crystal
if Dir.empty?(MIGRATIONS_DIR)
  exit! "no migrations to run"
else
  Micrate::Cli.run_up
end
```
Perhaps some error handling would also be good in addition to the conditional check.

Reply:
A general Exception is rescued at the bottom of the method, so I think a conditional here will do.

Reply:
I'd agree if this was an expected error. It's not, it's a bug. If we're going to add proper error handling, we'd want to check if the directory exists and contains any migrations. Since the app template ships with a `.keep` in `db/migrations` directory we would want to exclude that during our conditional checks as well.

But if we think we should handle this properly in Amber, I'd suggest we add proper error handling for all migration related commands. 🙂